src/index.js

```diff
@@ -177,48 +180,101 @@ class Saturn {
   }
 
   async _reportLogs () {
-    if (this.logs.length) {
-      await fetch(
-        'https://twb3qukm2i654i3tnvx36char40aymqq.lambda-url.us-west-2.on.aws/',
-        {
-          method: 'POST',
-          body: JSON.stringify({
-            bandwidthLogs: this.logs
-          })
-        }
-      )
-      this.logs = []
+    if (!this.logs.length) {
+      return
+    }
+
+    let bandwidthLogs = this.logs
+    if (this.hasPerformanceAPI) {
+      bandwidthLogs = this._matchLogsWithPerformanceMetrics(bandwidthLogs)
     }
+
+    await fetch(
+      this.opts.logURL,
+      {
+        method: 'POST',
+        body: JSON.stringify({ bandwidthLogs, logSender: this.opts.logSender })
+      }
+    )
+
+    this.logs = []
+    this._clearPerformanceBuffer()
+  }
+
+  /**
+   *
+   * @param {Array<object>} logs
+   */
+  _matchLogsWithPerformanceMetrics (logs) {
+    const logsToReport = []
+
+    for (const log of logs) {
+      const metrics = this._getPerformanceMetricsForLog(log)
+
+      if (!metrics.isFromBrowserCache) {
+        delete metrics.isFromBrowserCache
```

Comment:
What is the purpose of `delete metrics.isFromBrowserCache` if its undefined / null ?

Reply:
it could be `false`, so it'll end up in the log as `{ isFromBrowserCache: false }`. I'm trying to reduce the size of the log payload as much as possible as it's getting quite hefty.